src/navier_stokes/INSStaggeredCenteredConvectiveOperator.cpp

```diff
@@ -229,9 +230,9 @@ INSStaggeredCenteredConvectiveOperator::INSStaggeredCenteredConvectiveOperator(
     const std::string& object_name,
     Pointer<Database> input_db,
     const ConvectiveDifferencingType difference_form,
-    const std::vector<RobinBcCoefStrategy<NDIM>*>& bc_coefs)
+    std::vector<RobinBcCoefStrategy<NDIM>*> bc_coefs)
```

Comment:
And also `std::string object_name`?

(Ad infinitum. Maybe I will stop here on this particular comment...)